### PR TITLE
Add host to settings documentation

### PR DESCRIPTION
**Pull request by delenamalan**

### CODE REVIEW

File: docs/settings.rst [hunk 1]
```diff
@@ -70,6 +70,14 @@ proxy before forwarding along. Failure to strip these headers before sending to
 will result in an ``InvalidSignatureException`` due to request signing.
 
 
+host
+------
+
+Default: automatically constructed by boto.
```
Comment (ikonst) [inline docs/settings.rst:76]:
```suggestion
Default: automatically constructed by boto to account for region
```

Comment (ikonst) [reply docs/settings.rst:76]: (is this correct?)

Comment (delenamalan) [reply docs/settings.rst:76]: I'm not 100% sure. Maybe best to keep it vague?

Comment (ikonst) [reply docs/settings.rst:76]: The URLs are e.g. dynamodb.us-east-1.amazonaws.com, so I think that's the case

**Pull request merged**